gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -94,6 +93,10 @@ fluid.defaults("gpii.flowManager", {
             route: "/user/:userToken/logout",
```

Comment:
We can't in the long term maintain both the old "stateless" endpoints and the new "stateful" ones. We need to add comments to the endpoints and the JIRA that describes our migration plan and its milestones for how we will get rid of all the old endpoints and their names. It should clearly itemise i) what work needs to be done in the universal repo itself to allow the old endpoints to be destroyed ii) all the user listeners that will need to be updated, and how we will be able to run a testing plan that verifies them against the new system, iii) documentation updates and community dissemination we will need to do about the changes iv) how we will know when it is ok to sign off on deleting the old endpoints